src/elements/content-uploader/UploadsManagerAction.js.flow

```diff
@@ -28,11 +33,11 @@ const UploadsManagerAction = ({ onClick, hasMultipleFailedUploads }: Props) => {
 
     return (
         <div className="bcu-uploads-manager-action">
-            <PrimaryButton onClick={handleResumeClick} type="button" {...resin}>
-                <FormattedMessage {...resumeMessage} />
```

Comment:
After wrapping <FormattedMessage /> inside of blueprint's Button component, it will throw error below. Therefore using intl.formatMessage instead
`"This JSX tag's 'children' prop expects type 'ReactNode & (string | string[])' which requires multiple children, but only a single child was provided."`